Resources/views/base.html.twig

```diff
@@ -51,9 +51,9 @@
 
       <section id="main" role="main">
         <div class="header-subbar">
-          <div class="notifications-wrapper">
-            {{ include('@SumoCodersFrameworkCore/notifications.html.twig') }}
-          </div>
+          <div class="toasts-wrapper" aria-live="polite" aria-atomic="true">
```

Comment:
zijn die aria's hier wel nodig? Ze staan toch op elke toast apart? + de aria live is niet voor elke toast gelijk, dus idealiter zou die dan moeten updaten en wat dan als er meerdere toast met verschillende live's zijn? Ze lijken mij hier eigenlijk gewoon overbodig.